src/Interface/InterfacesViewContainer.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { isEqual, get } from 'lodash';
+import { Icon } from '@folio/stripes-components';
```

Comment:
code style

Reply:
still

Reply:
the same as above, this changes are in git branch